### DISCUSSION TITLE
Export multiple rabbitmq cluster in the Same deployment

### DISCUSSION THREAD

**Discussion by ItayNero** [kbudde/rabbitmq_exporter#367] [Q&A] [answered]
I am using your rabbitmq exporter on 1 rabbitmq cluster and it's running on a docker, I would like to configure it to export metrics on another rabbitmq cluster.
Is it possible?
Thank you🙏

**Comment by kbudde** [comment, marked as answer]
Hi, originally this exporter was meant to export data from a rabbitmq node. Support for is just working accidentally 😁

Export from multiple clusters is out of scope. Just run a second instance for the 2nd rabbitmq cluster.

**Reply by ItayNero** [reply]
But your code is open for extension so I can add to it to run on multiple clusters?

**Reply by kbudde** [reply]
No, please don't do it :)

The original scope of this exporter was to have one exporter per rabbit node.
Some people were using the exporter on a cluster and some changes were causing issues with this set up. Therefore the support for clusters (LB) was added. This already introduced some special cases.

Adding now another level to support multiple clusters with multiple nodes, would just complicate the exporter.

What's the reason that you want to have multi cluster support backed into the exporter? The resource usage should be the same.
